docs/src/testing/testing-with-rust.md

```diff
@@ -29,34 +29,34 @@ Our `tests/harness.rs` file could look like:
 <!--TODO add test here once examples are tested-->
 
 ```rust,ignore
-use fuel_tx::Salt;
+use fuel_tx::{ContractId, Salt};
```

Comment:
Hopefully we won't have to update this manually here in the future after the examples are tested with test harnesses.